README.md

```diff
@@ -1,17 +1,62 @@
-# Flex
+<br>
+<div align="center">
+  <img src="assets/valiot-logo-blue.png" alt="Valiot Logo" width="192" height="57" />
+</div>
+<br>  
 
-A WIP toolkit for fuzzy logic, this library includes functions for creating fuzzy variables, fuzzy sets, rules and a FLS.
+# FLex
 
-**TODO: Refactor.**
+A toolkit for fuzzy logic, this library includes functions to make fuzzy sets, variables and rules for creating a Fuzzy Logic System (FLS).
 
-## Usage
+The goal of the FLex is to easily design and efficiently operate fuzzy logic controllers without relying on external libraries.
```

Comment:
"The goal of FLex", sugeriría yo. A menos que sea por "The goal of the 'Fuzzy Logic System' ".